Description: kill off ijs-config
 The custom ijs-config tool gives different include paths than pkg-config
 (-I${INCLUDE_PATH}/ijs vs. -I${INCLUDE_PATH}/),
 which can be confusing for developers using this library.
 pkg-config is more standard anyway,
 so just kill the custom tool.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=7a7588c
Author: Brian Norris <computersforpeace@gmail.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=696528
Last-Update: 2019-08-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.am
+++ b/Makefile.am
@@ -47,8 +47,6 @@
 
 bin_PROGRAMS = ijs_client_example ijs_server_example
 
-bin_SCRIPTS = ijs-config
-
 
 libijs_la_SOURCES = \
 	ijs.c \
@@ -80,12 +78,9 @@
 
 pkgconfigdata_DATA = ijs.pc
 
-man_MANS = ijs-config.1
-
 EXTRA_DIST = \
 	common.mak unix.mak windows.mak \
-	ijs_spec.sgml state.eps state.fig ijs_spec.pdf \
-	$(man_MANS)
+	ijs_spec.sgml state.eps state.fig ijs_spec.pdf
 
 ## Rules
 
@@ -119,7 +114,7 @@
 	ltmain.sh
 
 CLEANFILES = gmon.out core ijs_spec.log ijs_spec.tex ijs_spec.aux ijs_spec.ps ijs_spec.pdf
-DISTCLEANFILES = ijs-config config.cache config.log config.status
-# ijs-config.in ijs.pc ijs.pc.in ijs.m4 ijs.h
+DISTCLEANFILES = config.cache config.log config.status
+# ijs.pc ijs.pc.in ijs.m4 ijs.h
 MAINTAINERCLEANFILES = INSTALL aclocal.m4 config.h.in configure Makefile.in stamp-h.in $(CLEANSCRIPTS)
 
--- a/configure.ac
+++ b/configure.ac
@@ -62,8 +62,8 @@
 dnl IJS_LIBS:    libraries for linking programs. ONLY to be used
 dnl                    to generate ${ijs_libs}
 dnl IJS_DEPLIBS: libraries for linking libraries against
-dnl ijs_cflags:  cflags to store in ijs-config
-dnl ijs_libs:    libs to store in ijs-config
+dnl ijs_cflags:  cflags to store for pkg-config
+dnl ijs_libs:    libs to store for pkg-config
 dnl LIBIJS_LIBS  libs to link programs IN THIS PACKAGE ONLY against
 
 dnl initial default values
@@ -160,6 +160,5 @@
 
 dnl output files and headers
 AC_CONFIG_FILES([Makefile])
-AC_CONFIG_FILES([ijs-config], [chmod +x ijs-config])
 AC_CONFIG_FILES([ijs.pc])
 AC_OUTPUT
--- a/ijs-config.1
+++ /dev/null
@@ -1,63 +0,0 @@
-.\"                                      Hey, EMACS: -*- nroff -*-
-.\" First parameter, NAME, should be all caps
-.\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
-.\" other parameters are allowed: see man(7), man(1)
-.TH IJS-CONFIG 1 "5 Feb, 2004"
-.\" Please adjust this date whenever revising the manpage.
-.\"
-.\" Some roff macros, for reference:
-.\" .nh        disable hyphenation
-.\" .hy        enable hyphenation
-.\" .ad l      left justify
-.\" .ad b      justify to both left and right margins
-.\" .nf        disable filling
-.\" .fi        enable filling
-.\" .br        insert line break
-.\" .sp <n>    insert n+1 empty lines
-.\" for manpage-specific macros, see man(7)
-.SH NAME
-ijs-config \- script to get information about the installed version of ijs
-.SH SYNOPSIS
-.B ijs-config
-[\-\-prefix\fI[=DIR]\fP] [\-\-exec\-prefix\fI[=DIR]\fP] [\-\-version] [\-\-libs] [\-\-cflags]
-.SH DESCRIPTION
-.PP
-\fIijs-config\fP is a program that is used to determine the compiler and
-linker flags that should be used to compile and link programs that use
-\fIIJS\fP.
-.SH OPTIONS
-This program follows the usual GNU command line syntax, with long
-options starting with two dashes (`-').
-A summary of options is included below.
-.TP
-.B \-\-help
-Show summary of options.  This may list additional advanced package
-configuration options not detailed in this manual page.
-.TP
-.B \-\-version
-Show installed libijs version.
-.TP
-.B  \-\-libs
-Print the linker flags that are necessary to link an \fIIJS\fP program.
-.TP
-.B  \-\-cflags
-Print the compiler flags that are necessary to compile an \fIIJS\fP program.
-.TP
-.B  \-\-prefix=PREFIX
-If specified, use PREFIX instead of the installation prefix that \fIIJS\fP
-was built with when computing the output for the \-\-cflags and
-\-\-libs options. This option is also used for the exec prefix
-if \-\-exec\-prefix was not specified. This option must be specified
-before any \-\-libs or \-\-cflags options.
-.TP
-.B  \-\-exec\-prefix=PREFIX
-If specified, use PREFIX instead of the installation exec prefix that
-\fIIJS\fP was built with when computing the output for the \-\-cflags
-and \-\-libs options.  This option must be specified before any
-\-\-libs or \-\-cflags options.
-.SH SEE ALSO
-.BR gtk-config (1)
-.SH AUTHOR
-This manual page was written by Roger Leigh
-\fI<rleigh@debian.org>\fP, originally for
-the Debian GNU/Linux system.
--- a/ijs-config.in
+++ /dev/null
@@ -1,56 +0,0 @@
-#!/bin/sh
-
-prefix=@prefix@
-exec_prefix=@exec_prefix@
-exec_prefix_set=no
-
-usage="\
-Usage: ijs-config [--prefix[=DIR]] [--exec-prefix[=DIR]] [--version] [--libs] [--cflags]"
-
-if test $# -eq 0; then
-      echo "${usage}" 1>&2
-      exit 1
-fi
-
-while test $# -gt 0; do
-  case "$1" in
-  -*=*) optarg=`echo "$1" | sed 's/[-_a-zA-Z0-9]*=//'` ;;
-  *) optarg= ;;
-  esac
-
-  case $1 in
-    --prefix=*)
-      prefix=$optarg
-      if test $exec_prefix_set = no ; then
-        exec_prefix=$optarg
-      fi
-      ;;
-    --prefix)
-      echo $prefix
-      ;;
-    --exec-prefix=*)
-      exec_prefix=$optarg
-      exec_prefix_set=yes
-      ;;
-    --exec-prefix)
-      echo $exec_prefix
-      ;;
-    --version)
-      echo @IJS_VERSION@
-      ;;
-    --cflags)
-      includes=-I@includedir@/ijs
-      echo $includes
-      ;;
-    --libs)
-      libdirs=-L@libdir@
-      echo $libdirs -lijs
-      ;;
-    *)
-      echo "${usage}" 1>&2
-      exit 1
-      ;;
-  esac
-  shift
-done
-
